new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiConfigMapOverride.java

```diff
@@ -0,0 +1,669 @@
+            .append(" \"value\": \"2\"")
+            .append(" }]");
+    logger.log(Level.INFO, "RestartVersion Patch String: {0}", patchStr);
+
+    patch = new V1Patch(new String(patchStr));
+    boolean rvPatched = assertDoesNotThrow(() -> 
+        patchDomainCustomResource(domainUid,domainNamespace,patch,"application/json-patch+json"),
+        "patchDomainCustomResource(restartVersion)  failed ");
+    assertTrue(rvPatched, "patchDomainCustomResource(restartVersion) failed");
+    logger.info("Snooze for 60 secs for Introspector to kick off");
+    try { 
+      Thread.sleep(60000);
+    } catch (java.lang.InterruptedException ie) {
+      logger.info("Got InterruptedException during Thread.sleep");
+      fail("Got InterruptedException during Thread.sleep");
+    }
+    // Verify a rolling restart is triggered in a sequential fashion 
+    // admin-server --> managed-server1 --> managed-server2 
+    logger.info("Check admin server pod {0} to be restarted in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodDeleted(adminServerPodName);
+    checkPodCreated(adminServerPodName);
+    checkPodRunning(adminServerPodName);
+    checkServiceCreated(adminServerPodName);
+
+    // check managed server pods re-started sequentially
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed server pod {0} to be restarted in namespace {1}",
+                        managedServerPrefix + i, domainNamespace);
+      checkPodDeleted(managedServerPrefix + i);
+      checkPodRunning(managedServerPrefix + i);
+      checkServiceCreated(managedServerPrefix + i);
+    }
+
+    oracle.weblogic.kubernetes.utils.ExecResult result = null; 
+    try {
+      String hostname = null;
+      if (System.getenv("K8S_NODEPORT_HOST") == null) {
+        hostname = InetAddress.getLocalHost().getHostName();
+      } else {
+        hostname = System.getenv("K8S_NODEPORT_HOST");
+      }
+      checkJdbc =  new StringBuffer("status=$(curl --user weblogic:welcome1 ");
+      checkJdbc.append("http://" + hostname + ":30711")
+             .append("/management/wls/latest/datasources/id/TestDataSource/")
+             .append(" -o /dev/null")
+             .append(" -w %{http_code});")
+             .append("echo ${status}");      
+      logger.info("CURL command {0}", new String(checkJdbc));
+      result = exec(new String(checkJdbc),true);
+    } catch (java.net.UnknownHostException uhe) {
+      logger.info("UnknownHostException is received {0}", uhe);
+      fail("Got UnknownHostException");
+    } catch (Exception ex) {
+      logger.info("Unexpected exception  {0}", ex);
+      fail("Got unexpected exception");
+    }
+    logger.info("Curl command returns {0}", result.toString());
+    assertEquals("200",result.stdout(),"Datasource configuration not found");
+    logger.info("Found the DataSource configuration ");
+  }
+
+  @AfterEach
+  public void tearDown() {
```

Comment:
don't need this - the extension will do it

Reply:
Small clarification - 
AfterEach is needed if multiple tests are in a class and it needs to reset things for the next test. 
If only one test is in the class, its not needed. Extension will cleanup everything after the last test is done.

Reply:
done